redis/cluster.py

```diff
@@ -1155,29 +1133,21 @@ def _execute_command(self, target_node, *args, **kwargs):
                     self.nodes_manager.update_moved_exception(e)
                 moved = True
             except TryAgainError:
-                log.exception("TryAgainError")
-
                 if ttl < self.RedisClusterRequestTTL / 2:
                     time.sleep(0.05)
             except AskError as e:
-                log.exception("AskError")
-
                 redirect_addr = get_node_name(host=e.host, port=e.port)
                 asking = True
             except ClusterDownError as e:
-                log.exception("ClusterDownError")
                 # ClusterDownError can occur during a failover and to get
                 # self-healed, we will try to reinitialize the cluster layout
                 # and retry executing the command
                 time.sleep(0.25)
                 self.nodes_manager.initialize()
                 raise e
             except ResponseError as e:
-                message = e.__str__()
-                log.exception(f"ResponseError: {message}")
                 raise e
```

Comment:
Is it usefull to keep the catching block for this exception? As we are catching it then re-raise it?

Reply:
I guess we could remove it!

Reply:
I get your point, nothing is really happening in this block? Would you be happy for me to remove it?

Reply:
I guess we can remove it unless we are expecting to see that the `try` block can raise a ResponseError exception.

Reply:
I think we need to be careful with that because when it catch BaseException it disconnects - I'm not sure that disconnecting is the expected behaviour in ResponseError

Reply:
You're right, nice catch. I didn't see that and ResponseError is a child of BaseException. Indeed, we can let this code as it was. I'm closing this discussion.